### ISSUE TITLE
[update:execute] command fails on D9

### ISSUE THREAD

**Issue by cburschka** [hechoendrupal/drupal-console#4302]
On the latest version (1.9.7), `drupal update:execute` will cause the following exception running on a D9 site:

```
Error: Call to undefined function Drupal\Console\Command\Update\update_fix_compatibility() in .../vendor/drupal/console/src/Command/Update/ExecuteCommand.php on line 130 #0 .../vendor/symfony/console/Command/Command.php(255): Drupal\Console\Command\Update\ExecuteCommand->execute()
#1 .../vendor/symfony/console/Application.php(1027): Symfony\Component\Console\Command\Command->run()
#2 .../vendor/symfony/console/Application.php(273): Symfony\Component\Console\Application->doRunCommand()
#3 .../vendor/drupal/console-core/src/Application.php(187): Symfony\Component\Console\Application->doRun()
#4 .../vendor/drupal/console/src/Application.php(64): Drupal\Console\Core\Application->doRun()
#5 .../vendor/symfony/console/Application.php(149): Drupal\Console\Application->doRun()
#6 .../vendor/drupal/console/bin/drupal.php(89): Symfony\Component\Console\Application->run()
#7 .../vendor/drupal/console/bin/drupal(4): require('...')
#8 {main}
Error: Call to undefined function Drupal\Console\Command\Update\update_fix_compatibility() in Drupal\Console\Command\Update\ExecuteCommand->execute() (line 130 of .../vendor/drupal/console/src/Command/Update/ExecuteCommand.php).
```

It appears that the function `update_fix_compatibility()` was deprecated in Drupal 8.8.5 and removed in Drupal 9.0.0: https://www.drupal.org/node/3026100

**Comment by geroldkl** [comment]
This also happens for the `drupal debug:update` command.

```
Error: Call to undefined function Drupal\Console\Command\Debug\update_fix_compatibility() in /var/www/vendor/drupal/console/src/Command/Debug/UpdateCommand.php on line 66 #0 /var/www/vendor/symfony/console/Command/Command.php(255): Drupal\Console\Command\Debug\UpdateCommand->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#1 /var/www/vendor/symfony/console/Application.php(1027): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 /var/www/vendor/symfony/console/Application.php(273): Symfony\Component\Console\Application->doRunCommand(Object(Drupal\Console\Command\Debug\UpdateCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 /var/www/vendor/drupal/console-core/src/Application.php(187): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 /var/www/vendor/drupal/console/src/Application.php(64): Drupal\Console\Core\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#5 /var/www/vendor/symfony/console/Application.php(149): Drupal\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 /var/www/vendor/drupal/console/bin/drupal.php(89): Symfony\Component\Console\Application->run()
#7 /var/www/vendor/drupal/console/bin/drupal(4): require('/var/www/vendor...')
#8 {main}
Error: Call to undefined function Drupal\Console\Command\Debug\update_fix_compatibility() in Drupal\Console\Command\Debug\UpdateCommand->execute() (line 66 of /var/www/vendor/drupal/console/src/Command/Debug/UpdateCommand.php).
```

**Comment by jenswegar** [comment]
Attached pull request that should fix the problem. Please test if it helps in your setup as well.

**Comment by LOBsTerr** [comment]
Fixed in 1.9.8
